pwndbg/gdblib/typeinfo.py

```diff
@@ -25,11 +25,22 @@ def lookup_types(*types):
 
 
 def update():
+    # Workaround for Rust stuff, see https://github.com/pwndbg/pwndbg/issues/855
+    restore_lang = gdb.execute("show language", to_string=True)
```

Comment:
nit: can we not use the same variable for the result of the `gdb.execute` and the language we assign to `restore_lang` later?

Reply:
Sure